dockerfiles/theia-dev/Dockerfile

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2018 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
+
+###
+# Builder Image
+#
+FROM node:8.12.0-alpine
+
+RUN apk add --update --no-cache curl jq expect make gcc g++ python git bash
+
+ENV HOME=/home/theia-dev
+WORKDIR ${HOME}
+
+# add eclipse che theia generator
+RUN yarn global add @eclipse-che/theia-generator@0.0.1-1543478099
+# change permissions
+RUN find ${HOME} -exec sh -c "chgrp 0 {}; chmod g+rwX {}" \; 2>log.txt
```

Comment:
Why do we need that log?

Reply:
+1, maybe /dev/null?

Reply:
@mmorhun mainly it's to avoid to see some output during the build of the image
and in case the instruction may have failed we can still get logs

I'm fine with dev null as well

Reply:
You may use `>/dev/null 2>&1` for suppressing everything from a command

Reply:
I would like to show all the outputs when building the image